src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -6769,6 +6835,7 @@ public sealed partial class FieldOffsetAttribute : System.Attribute
     }
     public partial struct GCHandle
     {
+        private object _dummy;
```

Comment:
I think this should be `int`. GCHandle does not have instance object fields.

Reply:
this might be a side-effect of me not filtering out statics while doing the contains reference check. I will regenerate once I update the tool.

Reply:
@jkotas the GCHandle holds a IntPtr which in turn holds a void* so based on our logic I would expect this to have an object dummy field. Is there something I'm missing?

Reply:
`void*` should be treated as `int`, not as `object`.

Reply:
OK with that change we get a bunch of updates. Can you have a look over https://github.com/dotnet/corefx/pull/26264/commits/9288c499b2491a2208225b8621ac99f7a301081e to ensure they look as expected?